### DISCUSSION TITLE
这个可以运行在cloudflare Page上面嘛

### DISCUSSION THREAD

**Discussion by uxiaohan** [spencerwooo/onedrive-vercel-index#163] [Q&A] [answered]
这个可以运行在cloudflare Page上面嘛

**Comment by spencerwooo** [comment]
我觉得或许能，但本项目不会去做特殊支持的。

**Comment by myl7** [comment, marked as answer]
参见 https://github.com/spencerwooo/onedrive-vercel-index/discussions/96#discussioncomment-1303864
